.github/workflows/GHA-Scala-Functional-Tests.yaml

```diff
@@ -34,42 +32,32 @@ jobs:
           ref: ${{ github.event.inputs.BRANCH || env.default-branch }}
       - uses: gradle/wrapper-validation-action@v1.0.4
 
-      #NEW Install required JDKs
-      # Install 8
       - name: Set up Java 8
         # https://github.com/actions/setup-java
         uses: actions/setup-java@v2
         with:
           distribution: 'adopt'
           java-version: 8
-          cache: 'gradle'
```

Comment:
Cache doesn't work?

Reply:
We are switching to remote caching for GE